glam/api/views.py

```diff
@@ -329,43 +329,32 @@ def probes(request):
 @api_view(["POST"])
 def random_probes(request):
     n = request.data.get("n", 3)
-    channel = request.data.get("channel", "nightly")
-    process = request.data.get("process", "parent")
-    os = request.data.get("os", "Windows")
     try:
         n = int(n)
     except ValueError:
         n = 3
 
-    probe_ids = list(
-        Probe.objects.exclude(info__kind="string")
-        .exclude(info__kind="boolean")
-        .values_list("id", flat=True)
-    )
-    if n > len(probe_ids):
-        raise ValidationError("Not enough probes to select `n` items.")
-
-    shuffle(probe_ids)
     probes = []
 
-    for id in probe_ids:
-        probe = Probe.objects.get(id=id)
-        # do not proceed if the probe is a boolean scalar
-        if not (probe.info["type"] == "scalar" and probe.info["kind"] == "boolean"):
-            try:
-                aggregations = get_firefox_aggregations(
-                    request,
-                    probe=probe.info["name"],
-                    channel=channel,
-                    os=os,
-                    process=process,
-                    aggregationLevel="version",
-                )
-            except NotFound:
-                continue
-            if aggregations:
-                probes.append({"data": aggregations, "info": probe.info})
-            if n <= len(probes):
-                break
+    # Get a random list of 9 probes from the Desktop nightly table.
+    aggs = DesktopNightlyAggregationView.objects.raw("""
+        SELECT id, metric, histogram
```

Comment:
Do we not get total clients here?

Reply:
probably below, not here, but I am not getting all the info I got before from this.

Reply:
@hamilton Yeah it's not all the same data. Let me know what else you'd like and we can add it back. That was part of the process I thought we'd go over. Basically I was trying to streamline it based on what I see on the home page.